### PR TITLE
Add freeimage and workflow

### PR DESCRIPTION
**Pull request by carstene1ns**
The vcpkg library uses additional patches and has more dependencies. We need to check how much weight this adds and could run our own later.

### CODE REVIEW

**Comment by Ghabry** [comment]
Maybe move this in a ``build_tools.cmd`` so this isn't built by default? Though building freeimage and (soon) wxwidgets aren't that big.

----------

There are alot of dependencies but they are all small and compile fast so works for me.

I only suggest setting webp to core to get rid of alot of dependencies:

```
libwebp[core] freeimage[core]
```

jasper could be also set to core to avoid pulling in opengl but wxwidgets has opengl as a required dep so it doesn't matter...

---------

With WX:

```
libwebp[core] freeimage[core] wxwidgets[core]
```

File: windows/build.cmd [hunk 1]
```diff
@@ -10,12 +10,12 @@ vcpkg install --triplet x86-windows-static --recurse^
  libvorbis[core] libsndfile[core] wildmidi[core] libxmp[core]^
  speexdsp[core] mpg123[core] opusfile[core] fluidsynth-easyrpg[core]^
  inih[cpp] lhasa-easyrpg[core]^
- sdl2-image[core] icu-easyrpg[core] nlohmann-json[core] fmt[core]
+ freeimage[core] icu-easyrpg[core] nlohmann-json[core] fmt[core]
```
Comment (Ghabry) [inline windows/build.cmd:13]: sdl2[core] must be added as a dependency now. before was pulled in via image 😅

Comment (carstene1ns) [reply windows/build.cmd:13]: Makes sense, thanks!

**Comment by Ghabry** [comment]
(Please do not rebuild windows. Fmtlib has a bug and breaks the build)

**Pull request merged**